cmd/configure/createprofile/createprofile.go

```diff
@@ -0,0 +1,52 @@
+package createprofile
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var CreateProfileCmd *cobra.Command
+
+func init() {
+	CreateProfileCmd = &cobra.Command{
+		Use:   "create-profile",
+		Short: "Create profile",
+		Long:  "Creates a new profile. Use this to create a profile with a name, host, and token.",
+		RunE:  createProfileHandler,
+	}
+
+	_ = CreateProfileCmd.Flags().String("name", "", "Name of profile.")
+	_ = CreateProfileCmd.Flags().String("host", "", "Host for profile.")
+	_ = CreateProfileCmd.Flags().String("token", "", "Token for profile.")
+}
+
+func createProfileHandler(cmd *cobra.Command, args []string) error {
+	var err error
+	var name string
+	var host string
+	var token string
+
+	f, err := os.OpenFile("$HOME/.jira-helper.config", os.O_APPEND|os.O_WRONLY, 0644)
```

Comment:
I would suggest you to use the Go built in [os.UserHomeDir](https://pkg.go.dev/os#UserHomeDir) function to retrieve the home directory, so it's already cross compatible